### ISSUE TITLE
"Failed to open" errors for files removed from the project.

### ISSUE THREAD

**Issue by mwbuksas** [Andersbakken/rtags#799]
rtags is generating messages like:

Failed to open /home/me/.rtags/project/57580/usrs /some/file/that/has/been/removed.cpp (2) 87

These are all for files that no longer exist in the project. Everything still appears to work fine, however. Thought it might be of interest though.

**Comment by Andersbakken** [comment]
Yeah. It's a bit of a known issue. If you restart rdm the messages should
go away. Our model for which source-files are indexed needs some love quite
frankly. I've been meaning to get to a refactor of that code but it's a
little bit involved. Thanks for the report though.

Anders

**Comment by mwbuksas** [comment]
A restart appears to cleared it right up, possibly affecting the speed of look-ups, although it's hard to be sure.

Thanks!
